test/controllers/EvidenceDownloadController.test.ts

```diff
@@ -26,15 +27,19 @@ describe('EvidenceDownloadController', () => {
     const contentLength = 1000;
     const contentType = `application/json`;
 
-    const metadata: FileMetadata = {
+    const expectedDownloadErrorHeading = 'The file can not be downloaded at this moment';
+    const expectedDownloadErrorMessage = 'Please try again later';
+
+    const createMetadata: FileMetadata = {
```

Comment:
Name `createMetadata` sounds like function while in fact is is just object literal. Please rename so that is does not sound like function.